pkg/api/controller.go

```diff
@@ -3635,6 +3636,53 @@ func (c *Controller) GetLakeFSVersion(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (c *Controller) PostStatsEvents(w http.ResponseWriter, r *http.Request, body PostStatsEventsJSONRequestBody) {
+	ctx := r.Context()
+	user, err := auth.GetUser(ctx)
+	if err != nil {
+		writeError(w, r, http.StatusUnauthorized, ErrAuthenticatingRequest)
+		return
+	}
+
+	for _, statsEv := range body.Events {
+		if statsEv.Class == "" {
+			writeError(w, r, http.StatusBadRequest, "invalid value: class is required")
+			return
+		}
+
+		if statsEv.Name == "" {
+			writeError(w, r, http.StatusBadRequest, "invalid value: name is required")
+			return
+		}
+
+		if statsEv.Count < 0 {
+			writeError(w, r, http.StatusBadRequest, fmt.Sprint("invalid value: count must be a non-negative integer"))
```

Comment:
You need to remove the call to Sprint - you do not format anything - waste of call